sources/packages/backend/apps/queue-consumers/src/processors/schedulers/institution-integration/ece-response/_tests_/ece-response-integration.scheduler.e2e-spec.ts

```diff
@@ -395,9 +400,16 @@ describe(
       expectedResult.summary = [
         `Starting download of file ${confirmEnrolmentResponseFile}.`,
         `The file ${confirmEnrolmentResponseFile} has been deleted after processing.`,
+        "Total file parsing errors: 1",
+        "Total disbursements found: 0",
```

Comment:
Because we are notifying the institution, we believe it is always required to give a summary of what happened in every scenario including no errors. 
Because when there is no errors we might have skipped n number of records which is not part of SIMS and it needs to be sent to the integration contacts. 

These decisions/discussions happened in the course of development and I think I need to update this in the story and have devs in sync with what happened.